plugins/modules/packaging/language/gem.py

```diff
@@ -220,6 +240,10 @@ def install(module):
 
     cmd = get_rubygems_path(module)
     cmd.append('install')
+    if (module.params['norc']
+            and ver
+            and LooseVersion('.'.join(map(str, ver))) >= LooseVersion('2.5.2')):
```

Comment:
This works fine, but I think it's cleaner if all the version checks are tuple comparisons or all the version checks are  `LooseVersion` comparisons. If tuple comparisons I would get rid of that major variable and replace those checks with ver < (2, 0, 0). If `LooseVersion` comparisons I would just capture the whole version string, return it from `get_rubygems_version`, and compare against `LooseVersion(ver)` consistently. Right now there's extra code to compensate for doing the same thing in two different styles.

Reply:
Agreed. I'll update this part.